docs/Design.md

```diff
@@ -82,16 +82,20 @@ The server exposes only two APIs:
 - `/v1/fetch_key` - Handles a request for one or more derived keys and returns them if access is permitted by the associated package / policies. Each request must meet the following requirements:
     - Be signed by the user's address using `signPersonalMessage`. For details, see the [signed_message](https://github.com/MystenLabs/seal/tree/main/crates/key-server/src/signed_message.rs) format.
     - Include a valid PTB, which is evaluated against the `seal_approve*` rules. For PTB construction guidelines, see [valid_ptb](https://github.com/MystenLabs/seal/tree/main/crates/key-server/src/valid_ptb.rs).
-    - Provide an encryption key to encrypt the response. Encrypting the response ensures that only the requester (the initiator) can decrypt and access the returned keys.
+    - Provide an ephemeral encryption key to encrypt the response. Encrypting the response ensures that only the requester (the initiator) can decrypt and access the returned keys.
 
 See [crates/key-server](https://github.com/MystenLabs/seal/tree/main/crates/key-server/src/server.rs) for the implementation of the key server.
 
 ## User confirmation and sessions
-Decryption keys returned from the key server are returned directly to the caller, which is typically the dApp's web page. To ensure that dApps can access only keys explicitly approved by the user, the user must approve the key access request in their wallet. This approval is granted once per package and authorizes a `session key`. The session key allows the dApp to retrieve associated decryption keys for a limited time without requiring repeated user confirmations.
+Decryption keys returned from the key server are returned directly to the caller, which is typically the dApp's web page. To ensure that dApps can access only keys explicitly approved by the user, the user must approve the key access request in their wallet. This approval is granted once per package and authorizes a `session key` that can be used to retrieve associated decryption keys for a limited time without requiring repeated user confirmations.
```

Comment:
nit: this is actually written as desired per the google developer style guide in active voice :)